lib/TikZImage.pm

```diff
@@ -35,7 +35,8 @@ sub new {
 		tikzLibraries => '',
 		texPackages   => {},
 		addToPreamble => '',
-		ext           => 'png',
+		ext           => 'svg',
+		svgMethod     => 'pdf2svg',
```

Comment:
I missed this before.  The 'convertOptions' should be added here to ensure that they are defined for the object.  You are perhaps assuming that this is always used via PGtikz.pl, but that may not be the case.  This package can be used directly, in which case the defaults will not be set.